### ISSUE TITLE
Jammer performance issues

### ISSUE THREAD

**Issue by YetheSamartaka** [KyklopCZ/Kyklop_EW#24]
After using jammers, performance severely decreased. With 3 being activated at once, performance issues became very apparent. 2x Zeus and 21 players. It was SpecOp, so one guy was in USAF F-35A, three people were at the base, two controlled USAF drones and were providing aerial reconnaissance + flying over jammed area. The rest of the players were operating in the field. Performance issues became very apparend mainly in jammed areas.

Server logs unfortunately do not provide any error messages or anything related to kyk_ew. Player logs are still in the status of gathering, but that is not that sure.

Arma 3 version 2.00
No headless clients were used
CBA_A3 version from 22 Apr @ 4:23pm
ACE version 3.13.4
KYK_EW version from 16 Oct @ 11:45am

**Comment by KyklopCZ** [comment]
I can't think of anything in my mod that would cause performance issues specifically in a jammed area. From my testing, each cycle of my mod takes less than 1ms to complete with a cycle occuring only once per 5 seconds (by default). But if you have any ideas what could cause those issues, I'd like to hear them.

**Comment by YetheSamartaka** [comment]
We've conducted another mission (A bit smaller, 20 players and 2 Zeus) where only 1 jammer was activated at once. Performance issues were not present. All players, Zeuses and AIs were inside jammed area most of the mission.

**Comment by KyklopCZ** [comment]
Yeah, based on that, I don't think my mod is the reason for your issues or at least not the only one. There also shouldn't be any massive drop in performance with more jammers active at one time. The only effect this should have is a linear increase in required processing power on the server with each jammer active and even with 10 jammers it shouldn't really create a delay of more than a few miliseconds.
